src/presenter/main-presenter.js

```diff
@@ -82,29 +82,41 @@ export default class MainPresenter {
   }
 
   init = () => {
+    this.#showNoPointsMessage();
+    this.#disableNewEventButton();
     this.#newEventButton.addEventListener('click', this.#createNewEvent);
-    this.#createComposePresenter();
     this.#setNotifications();
-    this.#updateEvents();
+    this.#model.init();
   };
 
-  #updateEvents = () => {
+  #setNotifications = () => {
+    this.#model.addDeletePointListener((args) => this.#whenDeletePoint(args));
+    this.#model.addUpdatePointsListener((args) => this.#whenUpdatePoints(args));
+    this.#model.addLoaderListener((args) => this.#afterLoad(args));
+  };
+
+  #afterLoad = () => {
+    const { offers, destinations } = this.#model.loaderState;
+    if (offers.ok && destinations.ok) {
+      this.#whenUpdatePoints();
+    } else {
+      this.#noPointsMessageView.message = NoPointsMessages.RELOAD;
+    }
+  };
+
+  #whenUpdatePoints = () => {
```

Comment:
```suggestion
  #handleUpdatePoints = () => {
```

Это "специально" для снижения нагрузки на мозги.

если есть событие "PointsUpdated" то можно подписаться на него используя метод

```js
model.addPointsUpdatedListener (handlePointsUpdated)
```
или
```js
model.addEventListener ('PointsUpdated',handlePointsUpdated)
```
или
```js
modle.onPointsUpdated = handlePointsUpdated
```

а обработчик назвать `handlePointsUpdated`